kinder/pkg/kubeadm/config.go

```diff
@@ -170,7 +170,7 @@ nodeRegistration:
     node-ip: "{{ .NodeAddress }}"
 ---
 # no-op entry that exists solely so it can be patched
-apiVersion: kubeadm.k8s.io/v1beta2
+apiVersion: kubeadm.k8s.io/v1beta4
```

Comment:
With https://github.com/kubernetes/kubernetes/pull/119156, 
```
  kubeletExtraArgs:
    node-ip: "{{ .NodeAddress }}"
```

should be 

```
  kubeletExtraArgs:
  - name: node-ip
     value: "{{ .NodeAddress }}"
```

Reply:
fixed